Sources/Impl/PinLayoutImpl.swift

```diff
@@ -234,20 +230,58 @@ class PinLayoutImpl: PinLayout {
         left({ "all() left coordinate" })
         return self
     }
+
+    @discardableResult
+    func all(_ value: CGFloat) -> PinLayout {
+        top(value,  { "all(\(value)) top coordinate" })
+        bottom(value,  { "all(\(value)) bottom coordinate" })
+        left(value,  { "all(\(value)) left coordinate" })
+        right(value,  { "all(\(value)) right coordinate" })
+        return self
+    }
     
     @discardableResult
     func horizontally() -> PinLayout {
         right({ "horizontally() right coordinate" })
         left({ "horizontally() left coordinate" })
         return self
     }
+
+    @discardableResult
+    func horizontally(_ value: CGFloat) -> PinLayout {
+        left(value, { return "horizontally(\(value)) left coordinate" })
+        right(value, { return "horizontally(\(value)) right coordinate" })
+        return self
+    }
+
+    @discardableResult
+    func horizontally(_ percent: Percent) -> PinLayout {
+        left(percent, { return "horizontally(\(percent.description)) left coordinate" })
+        right(percent, { return "horizontally(\(percent.description)) right coordinate" })
+        return self
+    }
+
     @discardableResult
     func vertically() -> PinLayout {
         top({ "vertically() top coordinate" })
         bottom({ "vertically() bottom coordinate" })
         return self
     }
 
+    @discardableResult
+    func vertically(_ value: CGFloat) -> PinLayout {
+        top(value, { return "vertically(\(value)) top" })
+        bottom(value, { return "vertically(\(value)) bottom" })
+        return self
+    }
+
+    @discardableResult
+    func vertically(_ percent: Percent) -> PinLayout {
+        top(percent, { return "vertically(\(percent.description)) top" })
```

Comment:
Just  missing the "coordinate" word: "vertically(\(value)) top coordinate".
Also, for the bottom.